### DISCUSSION TITLE
How do I disable the change password screen - it comes up every time I try to log in

### DISCUSSION THREAD

**Discussion by PercyP** [YetiForceCompany/YetiForceCRM#15253] [How to] [answered]
Hi, 

is there a setting to disable the change password screen altogether. It comes up every time I try to login now as administrator. I have already tried changing the settings under System/Security/Passwords - Configuration. It was set to 0 so I set it for 1000 but ideally would like to switch this off. 

I have also changed the config/security.php 	

/** Changing the settings by the user is possible true/false */
	public static $CHANGE_LOGIN_PASSWORD = false;

But nothing works.

Is there a setting elsewhere that needs to be changed?

Kind regards

**Comment by vovpff** [comment]
Try to uncheck `Force to change password` for your admin user.

**Comment by PercyP** [comment]
Hi,

where would i change that? I have looked in main.php also security but there is no such option. I have set the configuration value back to zero but still it keeps forcing me to change password every time I log in.

**Comment by PercyP** [comment, marked as answer]
I found what you meant (I can see it under Users) however there is no option in the record to change it so had to go in to the database in users and change it to zero there. Hopefully that has sorted it!

Thanks for the heads up.

**Comment by PercyP** [comment]
Unfortunately it is still happening :(

Although I changed the value in the mysql users table it hasn't changed it in the user details within the system, it still has Yes under force password change and when the record is opened there is no option to change it there.

**Comment by Gazzxy** [comment]
This is a bain of my life atm. though it goes away when I uncheck the box. it gets stuck on if I have to reset anyone password

**Comment by PercyP** [comment]
Hi Gazzxy,

I reported the issue here along with a temp solution that is so far working: 

https://github.com/YetiForceCompany/YetiForceCRM/issues/15257

Hope it helps

**Reply by Gazzxy** [reply]
thanks ill give it a go
